storage/txcache/txListForSenderAsSortedMapItem.go

```diff
@@ -0,0 +1,81 @@
+package txcache
+
+import (
+	"math"
+
+	"github.com/ElrondNetwork/elrond-go/storage/txcache/maps"
+)
+
+type senderScoreParams struct {
+	count uint64
+	// Size is in bytes
+	size uint64
+	// Fee is in micro ERD
+	fee uint64
+	gas uint64
+	// Price is in micro ERD
+	minGasPrice uint32
+}
+
+// GetKey return the key
+func (listForSender *txListForSender) GetKey() string {
+	return listForSender.sender
+}
+
+func (listForSender *txListForSender) getLastComputedScore() uint32 {
+	return listForSender.lastComputedScore.Get()
+}
+
+// ComputeScore computes the score of the sender, as an integer 0-100
+func (listForSender *txListForSender) ComputeScore() uint32 {
+	score := uint32(listForSender.computeRawScore())
+	listForSender.lastComputedScore.Set(score)
+	return score
+}
+
+func (listForSender *txListForSender) computeRawScore() float64 {
+	fee := listForSender.totalFee.GetUint64()
+	gas := listForSender.totalGas.GetUint64()
+	size := listForSender.totalBytes.GetUint64()
+	count := listForSender.countTx()
+	minGasPrice := listForSender.cacheConfig.MinGasPriceMicroErd
+
+	return computeSenderScore(senderScoreParams{count: count, size: size, fee: fee, gas: gas, minGasPrice: minGasPrice})
+}
+
+func computeSenderScore(params senderScoreParams) float64 {
+	allParamsDefined := params.fee > 0 && params.gas > 0 && params.size > 0 && params.count > 0
+	if !allParamsDefined {
+		return 0
+	}
+
+	PPUMin := float64(params.minGasPrice)
```

Comment:
suggestion to add the math formula as a comment

Reply:
Added :)

Reply:
:+1: :1st_place_medal: Great comments!